lms/djangoapps/edxnotes/views.py

```diff
@@ -66,6 +68,32 @@ def edxnotes(request, course_id):
     return render_to_response("edxnotes/edxnotes.html", context)
 
 
+@require_GET
+@login_required
+def list_notes(request, course_id):
```

Comment:
I was referring to `list_notes` and `search_notes`. The later only takes an extra query_string as far as I understand this.